awscli/customizations/wizard/wizards/dynamodb/new-table.yml

```diff
@@ -95,6 +100,106 @@ plan:
         type: prompt
         description: Choose a customer managed CMK
         choices: existing_cmks
+        condition:
+          variable: encryption_settings
+          equals: customer_managed_cmk
+  preview:
+    shortname: Preview
+    description: Preview results
+    values:
+      line_brake_if_KMS_set:
+        type: template
+        value: |
+          {% if {encryption_settings} != default %}
+          \ {% endif %}
```

Comment:
Instead of adding this variable, I think we should make the last line a parameter that we know will have to be included and therefore should not include the backslash. For example, we can just make the last command related to determining the capacity provision:
```
             {% if {capacity_mode} == PROVISIONED %}
            --provisioned-throughput \
              'ReadCapacityUnits={read_capacity},WriteCapacityUnits={write_capacity}'
            {% endif %}
            {% if {capacity_mode} == PAY_PER_REQUEST %}
            --billing-mode PAY_PER_REQUEST
            {% endif %}
```
In general, the ordering of parameters is not too important and I think this makes the templating a logic a bit simpler.